src/app.tsx

```diff
@@ -16,17 +18,23 @@ class App extends React.Component<{}, AppState> {
 
     this.state = {
       acc: null,
+      accs: null,
     };
   }
 
   public componentDidMount() {
     // get all fields
-    Account.retrieve(`SELECT ${generateSelect(Object.values(Account.FIELDS))} FROM Account LIMIT 1`)
-    .then((accs) => this.setState({acc: accs[0]}));
+    Account.retrieve(`SELECT ${generateSelect(Object.values(Account.FIELDS))} FROM Account`)
+    .then((accs) => this.setState({accs}));
   }
 
   public render() {
-    return <AccountDisplay acc={this.state.acc} />;
```

Comment:
curious why you removed this? I would expect instead there to be some sort of conditional rendering which determines if Search vs Display is shown

Reply:
I see now that you've instead chosen to render `<AccountDisplay />` from the <SearchResults />` component... Refactor so it's conditionally rendered here